packages/components/loading/src/loading-provider.tsx

```diff
@@ -426,14 +426,14 @@ const PageComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       bg: ["white", "black"],
-      maxW: "md",
+      maxW: "24rem",
       p: "md",
       display: "flex",
       flexDirection: "column",
       justifyContent: "center",
       alignItems: "center",
-      gap: "sm",
-      rounded: "md",
+      gap: "0.5rem",
+      rounded: "0.375rem",
       boxShadow: ["lg", "dark-lg"],
```

Comment:
Please refer to `shadows.ts` for reference!
```suggestion
      boxShadow: ["0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",  "0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4)"],
```